web_app_deployment/roles/kubectl/templates/deploy_web_app.yml.j2

```diff
@@ -19,7 +19,19 @@ spec:
         image: {{ harbor_url }}/piezo-resources/web_app:{{ git_tag }}
         imagePullPolicy: Always
         ports:
-        - containerPort: 8888 # Port where the application is exposed on 
+        - containerPort: 8888 # Port where the application is exposed on
+        volumeMounts:
+        - name: {{ secrets_name }}
+          mountPath: "/etc/secrets/"
```

Comment:
I've put it in the configuration file only so that it can be changed for running in a Windows dev environment. Will update the wiki to make this clear